ts/src/bitmex.ts

```diff
@@ -1853,14 +1853,32 @@ export default class bitmex extends Exchange {
             'ordType': orderType,
             'text': brokerId,
         };
-        if ((orderType === 'Stop') || (orderType === 'StopLimit') || (orderType === 'MarketIfTouched') || (orderType === 'LimitIfTouched')) {
-            const stopPrice = this.safeNumber2 (params, 'stopPx', 'stopPrice');
-            if (stopPrice === undefined) {
-                throw new ArgumentsRequired (this.id + ' createOrder() requires a stopPx or stopPrice parameter for the ' + orderType + ' order type');
-            } else {
-                request['stopPx'] = parseFloat (this.priceToPrecision (symbol, stopPrice));
-                params = this.omit (params, [ 'stopPx', 'stopPrice' ]);
+        const customTriggerType = (orderType === 'Stop') || (orderType === 'StopLimit') || (orderType === 'MarketIfTouched') || (orderType === 'LimitIfTouched');
+        // support for unified trigger format
+        const triggerPrice = this.safeNumberN (params, [ 'triggerPrice', 'stopPx', 'stopPrice' ]);
+        if ((triggerPrice !== undefined) && !customTriggerType) {
+            params = this.omit (params, [ 'triggerPrice', 'stopPx', 'stopPrice' ]);
+            request['stopPx'] = parseFloat (this.priceToPrecision (symbol, triggerPrice));
+            const triggerIf = this.safeString (params, 'triggerIf');
+            this.checkRequiredArgument ('createOrder', triggerIf, 'triggerIf', [ 'above', 'below' ]);
+            this.checkRequiredArgument ('createOrder', side, 'side', [ 'buy', 'sell' ]);
```

Comment:
@ttodua we should use the side here that is always defined to guess the triggerIf, if not provided, instead of explicitly requiring it